docs/_sidebar.md

```diff
@@ -2,11 +2,14 @@
 
 -   [**Getting Started**](getting-started)
 -   [**Provider**](provider)
+-   [**AlertsProvider**](alertsProvider)
```

Comment:
We don't actually export the service Providers (`Provider` is an aggregate of all of them), so no need to add this here!

```suggestion
```